share/normalize/utils.py

```diff
@@ -36,3 +36,7 @@ def format_address(self, address1='', address2='', city='', state_or_province=''
         return '{}, {}'.format(city, state_or_province)
 
     return address1
+
+
+def str_to_int(amount):
```

Comment:
Ah, I did consider that. It probably would be better since all normalizers adding amount to award will have to convert the string to an int. Will update.